dataplane/saiserver/saiserver.go

```diff
@@ -233,5 +235,12 @@ func New(ctx context.Context, mgr *attrmgr.AttrMgr, s *grpc.Server, opts *dplane
 	saipb.RegisterSystemPortServer(s, srv.systemPort)
 	saipb.RegisterTamServer(s, srv.tam)
 
+	m := otel.GetMeterProvider().Meter("openconfig/lemming/dataplane/saiserver")
+	c, err := m.Int64Counter("lucius-instance")
+	if err != nil {
+		return nil, err
+	}
+	c.Add(ctx, 1)
```

Comment:
I suppose this increases the counter by 1, but why do we do that?

Reply:
we can aggregate all these counters to watch usage